src/storage/src/error.rs

```diff
@@ -81,7 +82,16 @@ pub enum Error {
         location: Location,
         source: object_store::Error,
     },
-
+    #[snafu(display("Fail to compress object by {}, source: {}", compress_type, err_msg))]
```

Comment:
Add a new line between error variants.
```suggestion

    #[snafu(display("Fail to compress object by {}, source: {}", compress_type, err_msg))]
```